serving/docker/aarch64.Dockerfile

```diff
@@ -27,12 +27,12 @@ CMD ["serve"]
 
 COPY scripts scripts/
 RUN mkdir -p /opt/djl/conf
```

Comment:
did you removed:
`ENV MODEL_SERVER_HOME=/opt/djl` ?

Reply:
It is in line 23